### ISSUE TITLE
[Bug] Fix SearchView Accessibility to read out Semantic Hint

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#1228]
Once this issue is addressed

https://github.com/dotnet/maui/issues/1227

we'll need to fix how accessibility properties get applied to search view

**Comment by rachelkang** [comment]
It seems like we're ready for this, and from what I can tell, the description and hint text are not read aloud.

**Comment by v-longmin** [comment]
Thank you so much for your bug report! Unfortunately, we cannot reproduce your issue with the information above, could you provide a sample project and screenshots or more details that we can reproduce your issue? That should greatly speed up the process, thanks!

**Comment by PureWeen** [comment]
This is an internal tracking issue. Don't worry about validating this @v-longmin

**Comment by rachelkang** [comment]
On Android currently:
SearchBar semantics are looking mostly great. However, hint is never read aloud whatsoever

On iOS currently:
Hint seems to be assigned not to the actual editable part of the SearchBar, but the entire control as a whole. As a result, when hint is assigned, hint gets read aloud, but nothing else does and the editable textfield can't be accessed!!!

The root of the problem seems to be the same for both, that the hint is being assigned to the outer / overarching control, as opposed to just the editable text input area
